openeo/rest/job.py

```diff
@@ -264,6 +265,10 @@ def __init__(self, job: RESTJob):
     def __repr__(self):
         return "<JobResults for job {j!r}>".format(j=self._job.job_id)
 
+    def _repr_html_(self):
+        response = self.get_metadata()
```

Comment:
This is tricky:  `get_metadata()` will (currently) fail when the job is not finished yet (`requests to this endpoint MUST be rejected with openEO error JobNotFinished`) , you don't want this behavious in `_repr_html_`

Reply:
So I guess I should just catch the error and print something else? Or what would be better?

Reply:
I think I can improve the exception that will be thrown there, so that you can check HTTP status code and openeo Error code, then you can indeed render some kind of "job not finished yet" message

Reply:
aha probably no change required: you should already get a `OpenEoApiError` exception there with `.http_status_code` >= 400 and `.code` == "JobNotFinished"

Reply:
I've tried it, but I can't import OpenEoApiError in Job, seems like a recursive dependency. How to solve that?

Reply:
Shall I just check for OpenEoClientException instead?

Reply:
...but then I can't access the message

Reply:
Implemented, that was more complicated than I thought it would be...